client/components/mma/shared/assets/SwitchOffsetPaymentIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import { iconSize } from '@guardian/source-foundations';
+import type { IconProps } from '@guardian/source-react-components';
+
+export const SwitchOffsetPaymentIcon = ({ size }: IconProps) => (
+	<svg
+		width={size ? iconSize[size] : undefined}
+		height={undefined}
+		viewBox="-3 -3 30 30"
+		xmlns="http://www.w3.org/2000/svg"
+		focusable={false}
+	>
+		<path
+			fillRule="evenodd"
+			clipRule="evenodd"
+			d="M11.1061 21.9102C7.66081 21.9102 4.60595 20.1645 2.81438 17.5231L2.92922 16.9029L3.73314 16.4206L4.35329 16.5814C5.84627 18.6945 8.28097 20.0956 11.1061 20.0956C15.631 20.0956 19.352 16.3747 19.352 11.8039C19.352 7.25602 15.631 3.55803 11.1061 3.55803C8.51066 3.55803 6.28268 4.70647 4.74377 6.54398L8.07425 7.09524V8.24368H2.19422L1.75781 7.80727V1.74349H2.88329L3.45751 5.21179C5.29502 3.07568 8.00534 1.74349 11.1061 1.74349C16.6876 1.74349 21.2124 6.24539 21.2124 11.8039C21.2124 17.4083 16.6876 21.9102 11.1061 21.9102Z"
+		/>
+		<path d="M10.7671 16.5243C9.98136 16.4389 9.34936 16.2553 8.87109 15.9734V14.7052C9.17001 14.8504 9.49028 14.9699 9.83191 15.0639C10.1821 15.1578 10.5493 15.2261 10.9336 15.2688L11.3436 12.2967L10.908 12.1558C10.5749 12.0533 10.2846 11.9338 10.0369 11.7971C9.79774 11.6605 9.59277 11.4939 9.42196 11.2975C9.25969 11.1011 9.13585 10.8662 9.05045 10.5929C8.97358 10.3196 8.93515 9.99078 8.93515 9.60646C8.93515 9.23067 8.9992 8.88478 9.12731 8.56878C9.25542 8.24424 9.44331 7.96667 9.69099 7.73608C9.9472 7.50548 10.2632 7.32613 10.639 7.19802C11.0233 7.06991 11.4674 7.00586 11.9713 7.00586H12.0482L12.2147 5.81445H13.1371L12.9706 7.08272C13.2695 7.12543 13.5257 7.18521 13.7392 7.26208C13.9527 7.3304 14.1449 7.40726 14.3157 7.49267V8.76094C14.1193 8.67554 13.8929 8.59867 13.6367 8.53035C13.3805 8.45348 13.0987 8.3937 12.7912 8.351L12.4197 11.0797L12.6119 11.1309C12.962 11.242 13.2737 11.3658 13.547 11.5025C13.8289 11.6306 14.0637 11.7971 14.2516 12.0021C14.4395 12.1985 14.5804 12.4376 14.6744 12.7195C14.7769 12.9928 14.8281 13.3344 14.8281 13.7444C14.8281 14.5984 14.5634 15.2816 14.0339 15.7941C13.5129 16.298 12.7272 16.5585 11.6767 16.5755L11.4461 18.3562H10.5237L10.7671 16.5243ZM13.2652 13.8853C13.2652 13.7059 13.2481 13.5522 13.214 13.4241C13.1798 13.2874 13.12 13.1679 13.0346 13.0654C12.9492 12.9629 12.8382 12.8732 12.7015 12.7963C12.5734 12.7195 12.4112 12.6469 12.2147 12.5786L11.856 15.256C12.3343 15.2048 12.6887 15.0596 12.9193 14.8205C13.1499 14.5813 13.2652 14.2696 13.2652 13.8853ZM10.434 9.47835C10.434 9.83705 10.5109 10.1103 10.6646 10.2982C10.8269 10.4861 11.1173 10.6484 11.5357 10.7851L11.5486 10.7979L11.8816 8.29975C11.4119 8.31683 11.0532 8.42786 10.8055 8.63283C10.5579 8.83781 10.434 9.11964 10.434 9.47835Z" />
```

Comment:
One thing I forgot to say about SVGs is the output from design tools like Figma is generally not as optimal as it could be so it's always good to run them through SVGO to optimise the size and strip out redundant attributes and metadata. There's a handy online version courtesy of Jake Archibald: https://jakearchibald.github.io/svgomg/

(I'd double-check the optimised version below to make sure it still looks how it should though.)

```suggestion
		<path d="M11.106 21.91c-3.445 0-6.5-1.745-8.292-4.387l.115-.62.804-.482.62.16c1.493 2.114 3.928 3.515 6.753 3.515 4.525 0 8.246-3.721 8.246-8.292 0-4.548-3.721-8.246-8.246-8.246-2.595 0-4.823 1.148-6.362 2.986l3.33.551v1.149h-5.88l-.436-.437V1.743h1.125l.575 3.469c1.837-2.136 4.547-3.469 7.648-3.469 5.582 0 10.106 4.502 10.106 10.06A10.092 10.092 0 0 1 11.106 21.91Z"/>
		<path d="M10.767 16.524c-.786-.085-1.418-.269-1.896-.55v-1.269c.299.145.62.265.96.359.351.094.718.162 1.103.205l.41-2.972-.436-.141a4.612 4.612 0 0 1-.871-.359 2.354 2.354 0 0 1-.615-.5 2.054 2.054 0 0 1-.372-.704 3.68 3.68 0 0 1-.115-.987c0-.375.064-.721.192-1.037a2.25 2.25 0 0 1 .564-.833c.256-.23.572-.41.948-.538a4.212 4.212 0 0 1 1.332-.192h.077l.167-1.192h.922l-.166 1.269c.299.042.555.102.768.18.214.067.406.144.577.23V8.76a4.834 4.834 0 0 0-.68-.23 5.65 5.65 0 0 0-.845-.18l-.371 2.729.192.05c.35.112.662.236.935.372.282.129.517.295.705.5.188.196.328.436.422.717.103.274.154.615.154 1.025 0 .854-.265 1.538-.794 2.05-.521.504-1.307.764-2.357.782l-.23 1.78h-.923l.243-1.832Zm2.498-2.639c0-.18-.017-.333-.051-.46a.889.889 0 0 0-.18-.36 1.18 1.18 0 0 0-.333-.269 2.645 2.645 0 0 0-.486-.217l-.359 2.677c.478-.051.833-.196 1.063-.436.23-.239.346-.55.346-.935Zm-2.831-4.407c0 .359.077.632.23.82.163.188.453.35.872.487l.013.013.333-2.498c-.47.017-.829.128-1.076.333-.248.205-.372.487-.372.845Z"/>
```

Reply:
Great suggestion 👍  really useful